### PR TITLE
WP-581 - E.COM | As a reader I want to see comments without any number so I know where to comment

### PR DESCRIPTION
**Pull request by yasgreen93**
https://jira.economist.com/browse/WP-581

### CODE REVIEW

**Comment by yasgreen93** [comment]
@varholak-peter absolutely beautiful - refactored.

**Comment by varholak-peter** [comment]
@yasgreen93 I probably confused you a bit with my last comment on what I meant. :D
I tried to say that now since you removed plenty of code and one CSS selector you can remove it from css files as to prevent dead code. (If there is any)
There is no need to add more classes and you just introduced duplicity with your last commit so you can revert it.

Sorry for confusion. :D

**Comment by yasgreen93** [comment]
@varholak-peter Ah ok, well you actually helped me see a small error with the CSS on mobile anyway so this extra classname is needed.

File: src/parts/comments.js [hunk 1]
```diff
@@ -1,21 +1,14 @@
...
 export default function Comments({ firstToCommentLabel, commentCount, viewCommentsLabel, commentsUri }) {
-  let content = firstToCommentLabel;
-  let pre = null;
-  let post = null;
-  if (commentCount > 0) {
-    content = commentCount;
-    pre = <span className="blog-post__comments-contentwrapper">{`${ viewCommentsLabel } (`}</span>;
-    post = <span className="blog-post__comments-contentwrapper">{')'}</span>;
-  }
+  const className = commentCount > 0 ? 'blog-post__comments-contentwrapper' : '';
```
Comment (tomiberes) [inline src/parts/comments.js]: According to what I see you don't want to display `view comments` when you have comments? This doesn't makes much sense to me.
`blog-post__comments-contentwrapper` in `fe-blogs` is `display: none`.

**Comment by yasgreen93** [comment]
@tomiberes so the two bits of text that appear are either "Be the first to comment" or "View comments". On desktop and mobile "Be the first to comment" is always displayed, however with "View comments" this is only displayed on desktop and hidden on mobile. I had to have this ternary because of the differing scenarios. I have removed the "blog-post__comments-content" as no css was being applied to it.

**Comment by tomiberes** [comment]
LGTM as long as it works as intended.

**Pull request merged**